interop-testing/src/test/java/io/grpc/testing/integration/Http2OkHttpTest.java

```diff
@@ -88,8 +88,7 @@ protected ManagedChannel createChannel() {
         .overrideAuthority(GrpcUtil.authorityFromHostAndPort(
             TestUtils.TEST_SERVER_HOST, serverPort));
     try {
-      builder.sslSocketFactory(TestUtils.newSslSocketFactoryForCa(
-          TestUtils.loadCert("ca.pem")));
+      builder.sslSocketFactory(TestUtils.newSslSocketFactoryForCa(TestUtils.loadCert("ca.pem")));
```

Comment:
I started changing this, but realized netty wants a real file, and backed out most of this file.  I guess I missed this one.